apparmor.d/groups/kde/mariadbd_akonadi

```diff
@@ -0,0 +1,54 @@
+abi <abi/3.0>,
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/mariadb{,-*} @{bin}/mariadbd @{bin}/mysql @{bin}/mysqld{,*}
+profile mariadbd_akonadi @{exec_path} flags=(complain) {
+  include <abstractions/base>
+  include <abstractions/bash>
```

Comment:
The bash abstraction is only needed for interactive shell. So not here.

Reply:
That abstraction was in the original profile. I have to check the consequences once removed.